test/test_utils.py

```diff
@@ -90,7 +90,7 @@ def test_draw_boxes(self):
 
         path = os.path.join(os.path.dirname(os.path.abspath(__file__)), "assets", "fakedata", "draw_boxes_util.png")
         if not os.path.exists(path):
-            Image.fromarray(result.permute(1, 2, 0).numpy()).save(path)
+            write_png(result.permute(1, 2, 0), path)
```

Comment:
```suggestion
            write_png(result, path)
```
We no longer need to permute the channels with TorchVision.